h5p-bildetema/src/components/TopicGridElement/TopicGridElement.tsx

```diff
@@ -14,9 +15,15 @@ export const TopicGridElement: React.FC<TopicGridElementProps> = ({
   title,
   index,
 }) => {
+  const navigate = useNavigate();
+  const handleClick = (): void => {
+    // temp - stop navigation when the words are displayed instead of topics
+    if (item) return;
+    navigate(`${encodeURIComponent(title.toLowerCase().split(" ").join("-"))}`);
+  };
   return (
-    <button className={styles.gridElement} type="button">
-      <span>{`${index + 1}. ${title} ${item}`}</span>
+    <button className={styles.gridElement} type="button" onClick={handleClick}>
+      <span>{`${index + 1}. ${title}`}</span>
     </button>
```

Comment:
Can we not use the `Link` component here? It will produce an `a` tag instead of a `button`, which is better for accessibility.

Reply:
\+ Link apparently uses useNavugate inside but is wrapped with "smart" <a>, so yeah that's a good idea.